src/devices/SenseHat/SenseHatLedMatrixSysFs.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Device.Gpio;
+using System.Diagnostics;
+using System.Drawing;
+using System.Runtime.CompilerServices; 
+
+namespace Iot.Device.SenseHat
+{
+    public class SenseHatLedMatrixSysFs : SenseHatLedMatrix
+    {
+        private const string SenseHatDeviceName = "RPi-Sense FB";
+
+        // Pixel length in binary representation
+        private const int PixelLength = 2; // RGB565
+
+        private FileStream _deviceFile;
+
+        public SenseHatLedMatrixSysFs()
+        {
+            string device = GetSenseHatDevice();
+
+            if (device == null)
+            {
+                throw new InvalidOperationException("Sense HAT not found. Ensure device is enabled in config.txt.");
+            }
+
+            _deviceFile = new FileStream(device, FileMode.Open, FileAccess.ReadWrite);
+            Clear(Color.Black);
+        }
+
+        public override void Write(ReadOnlySpan<Color> colors)
+        {
+            if (colors.Length != NumberOfPixels)
+                throw new ArgumentException($"`{nameof(colors)}` must have exactly {NumberOfPixels} elements.");
+
+            StartWritingColors();
+
+            for (int i = 0; i < NumberOfPixels; i++)
+            {
+                WriteColor(colors[i]);
+            }
+
+            EndWriting();
+        }
+
+        public override void Clear(Color color = default(Color))
```

Comment:
Why does Clear take a Color? Should there be two separate APIs? `Clear()` and `FilI(Color)`?

Reply:
I changed to Fill but with default arg as is - Clear feels redundant